features/ubuntu_pro.feature

```diff
@@ -613,3 +613,300 @@ Feature: Command behaviour when auto-attached in an ubuntu PRO image
+                            "affordances": {"series": ["bionic", "focal"]}
+                        },
+                        {
+                            "type": "fips-updates",
+                            "affordances": {"series": ["bionic", "focal"]}
+                        }
+                    ]
+                }
+            }
+        }
+        """
+        And I create the file `/etc/ubuntu-advantage/uaclient.conf` with the following:
+        """
+        contract_url: 'https://contracts.canonical.com'
+        data_dir: /var/lib/ubuntu-advantage
+        log_level: debug
+        log_file: /var/log/ubuntu-advantage.log
+        """
+        And I append the following on uaclient config:
+        """
+        features:
+          machine_token_overlay: "/tmp/machine-token-overlay.json"
+        """
+        And I run `ua auto-attach` with sudo
+        And I run `ua status --wait` with sudo
+        Then stdout matches regexp:
+            """
+            SERVICE       ENTITLED  STATUS    DESCRIPTION
+            cc-eal        +yes +n/a  +Common Criteria EAL2 Provisioning Packages
+            """
+        Then stdout matches regexp:
+            """
+            esm-apps      +yes +enabled +UA Apps: Extended Security Maintenance \(ESM\)
+            esm-infra     +yes +enabled +UA Infra: Extended Security Maintenance \(ESM\)
+            fips          +yes +disabled +NIST-certified core packages
+            fips-updates  +yes +disabled +NIST-certified core packages with priority security updates
+            livepatch     +yes +enabled  +Canonical Livepatch service
+            usg           +yes +disabled +Security compliance and audit tools
+            """
+            When I run `ua enable <fips-service> --assume-yes` with sudo
+            Then stdout matches regexp:
+                """
+                Updating package lists
+                Installing <fips-name> packages
+                <fips-name> enabled
+                A reboot is required to complete install
+                """
+            When I run `ua status --all` with sudo
+            Then stdout matches regexp:
+                """
+                <fips-service> +yes                enabled
+                """
+            And I verify that running `apt update` `with sudo` exits `0`
+            And I verify that running `grep Traceback /var/log/ubuntu-advantage.log` `with sudo` exits `1`
+            When I run `apt-cache policy ubuntu-gcp-fips` as non-root
+            Then stdout does not match regexp:
+            """
+            .*Installed: \(none\)
+            """
+            When I reboot the `<release>` machine
+            And  I run `uname -r` as non-root
+            Then stdout matches regexp:
+                """
+                gcp-fips
+                """
+            When I run `cat /proc/sys/crypto/fips_enabled` with sudo
+            Then I will see the following on stdout:
+            """
+            1
+            """
```

Comment:
Just a minor formatting issue, an accidental identation.

Reply:
Sorry @CalvoM I think I am missing your point here

Reply:
@lucasmoura The 30 lines above this are all indented too much :)

Reply:
Oh, now I see it :facepalm: Thanks for spotting that @CalvoM and @orndorffgrant